pkg/jira/customfield.go

```diff
@@ -1,9 +1,10 @@
 package jira
 
 const (
-	customFieldFormatOption = "option"
-	customFieldFormatArray  = "array"
-	customFieldFormatNumber = "number"
+	customFieldFormatOption  = "option"
+	customFieldFormatArray   = "array"
+	customFieldFormatNumber  = "number"
+	customFieldFormatProject = "project"
```

Comment:
Hi @oveaurs, I don't see this field format in the [Jira doc](https://developer.atlassian.com/cloud/jira/service-desk/rest/intro/#fieldformats). I am wondering what's the datatype for this field returned by the [`/field`](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-fields/#api-rest-api-2-field-get) API?

Reply:
```{
    "id": "customfield_13467",
    "name": "Cross over project",
    "custom": true,
    "orderable": true,
    "navigable": true,
    "searchable": true,
    "clauseNames": [
      "cf[13467]",
      "Cross over project"
    ],
    "schema": {
      "type": "project",
      "custom": "com.atlassian.jira.plugin.system.customfieldtypes:project",
      "customId": 13467
    }
  },
```
I found a reference to that datatype here: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-fields/

Reply:
I see, thank you for the reference. The changes looks OK to me.